src/iosrv/init.c

```diff
@@ -418,6 +418,38 @@ set_abt_max_num_xstreams(int n)
 	return 0;
 }
 
+static int
+abt_thread_stacksize(void)
+{
+	char   *env;
+
+	env = getenv("ABT_THREAD_STACKSIZE");
+	if (env == NULL)
+		env = getenv("ABT_ENV_THREAD_STACKSIZE");
+	if (env != NULL)
+		return atoi(env);
+	return 0;
+}
+
+static int
+set_abt_thread_stacksize(int n)
+{
+	char   *name = "ABT_THREAD_STACKSIZE";
+	char   *value;
+	int	rc;
+
+	D_ASSERTF(n > 0, "%d\n", n);
+	D_ASPRINTF(value, "%d", n);
+	if (value == NULL)
+		return -DER_NOMEM;
+	D_INFO("Setting %s to %s\n", name, value);
+	rc = setenv(name, value, 1 /* overwrite */);
+	D_FREE(value);
+	if (rc != 0)
+		return daos_errno2der(errno);
+	return 0;
+}
+
```

Comment:
[Optional] Could be placed in libgurt to avoid duplication?

Reply:
Never mind, sorry. libgurt should not depend on Argobots. There's no good place that could be shared between iosrv and vos.